src/thematic/4_vegetation/sfp-401_vegopname-terrest_nl/06_materialen.Rmd

```diff
@@ -24,8 +24,7 @@
 
 -   voldoende schrijfmateriaal
 
--   flora (naamgeving volgens Flora van België het Groothertogdom Luxemburg,
-    > Noord Frankrijk en de aangrenzende gebieden (Lambinon *et al.*, 1998))
+-   flora [(]@RN157]
```

Comment:
```suggestion
-   flora [@RN157]
```

Reply:
@ElsDeBie als je akkoord bent met deze verandering kan je gewoon op de "commit suggestion" knop drukken. Dan wordt deze wijziging toegepast in deze branch op de remote.  Om de wijziging lokaal te krijgen moet je nog in Rstudio op de pull knop drukken